torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -129,9 +129,9 @@ void ONNXShapeTypeInference(Node* n, int opset_version) {
   // TODO: Some ops have conversion happen at Peephole pass.
   //       The conversion here is incomplete for these ops.
   //       e.g: ListConstruct, ListUnpack, etc.
-  std::string model_str;
+  onnx::ModelProto model_p;
```

Comment:
rename to `model_proto`, so line 147 can be removed.

Reply:
Changes made